auto-value-gson/src/main/java/com/ryanharter/auto/value/gson/AutoValueGsonExtension.java

```diff
@@ -671,7 +708,166 @@ private MethodSpec createReadMethod(ClassName className,
     return readMethod.build();
   }
 
-  /** Returns a default value for initializing well-known types, or else {@code null}. */
+  private MethodSpec createReadMethod(ClassName className,
```

Comment:
Let's maybe rename this one to "createReadMethodWithBuilder" for clarity?

Reply:
Good call, I found this annoying too.